frontend/src/components/header.js

```diff
@@ -1,33 +1,175 @@
-    />
-  </header>
-);
+const Header = () => {
+  const browseList = [
+    { title: "About", route: "/about" },
+    { title: "Connect", route: "/connect" },
+    { title: "Next Steps", route: "/next-steps" },
+    { title: "Events", route: "/events" },
+    { title: "Watch", route: "/watch" },
+    { title: "Give", route: "/give" },
+  ];
+
+  const locationsList = [
+    { title: "Ann Arbor", route: "/" },
+    { title: "Austin", route: "https://austin.hmcc.net/" },
+    { title: "Detroit", route: "https://detroit.hmcc.net/" },
+    { title: "Hong Kong", route: "https://hongkong.hmcc.net/" },
+    { title: "Jakarta", route: "https://jakarta.hmcc.net/" },
+  ];
+
+  const [isOpen, setIsOpen] = React.useState(false);
+  const path = window.location.pathname;
```

Comment:
here is only minore changes needes
```suggestion
  const path = typeof window !== "undefined" && window.location.pathname || "";
```